package.json

```diff
@@ -73,6 +73,8 @@
     "prettier": "^1.19.1",
     "raf": "^3.4.0",
     "raw-loader": "^0.5.1",
+    "react-dnd-test-backend": "^9.4.0",
+    "react-dnd-test-utils": "^9.4.0",
```

Comment:
I'm assuming these don't need to be packaged, please add them at https://github.com/theforeman/foreman-packaging/blob/rpm/develop/update-requirements#L10-L29

Reply:
Like this: theforeman/foreman-packaging#4376 ? Thanks, I didn't know it's necessary.

Reply:
it's not quite a blocker but just so the next one someone from @theforeman/packaging team syncs the foreman requirements it doesn't add this.